Python/pywarpx/picmi.py

```diff
@@ -2084,6 +2087,9 @@ def initialize_inputs(self):
             B_fields_list = ['Bx', 'By', 'Bz']
             J_fields_list = ['Jx', 'Jy', 'Jz']
             A_fields_list = ['Ax', 'Ay', 'Az']
+        # Je fields list if Hybrid solver is used.
+        if type(solver) == HybridPICSolver:
+            Je_fields_list = ['Jer', 'Jet', 'Jez']
```

Comment:
The `Je_fields_list` can always be set and doesn't need to depend on the solver type. This would avoid having to pass in `solver` since this is the only place it is used.